pkgs/applications/misc/blender/default.nix

```diff
@@ -75,7 +75,9 @@ stdenv.mkDerivation rec {
     '';
 
   cmakeFlags =
-    [ "-DWITH_MOD_OCEANSIM=ON"
+    [
+      "-DWITH_ALEMBIC=OFF"
```

Comment:
Given that this works fine on linux with alembic (according to the comment in commits message for ac560382c5fe69b8b5375a44117da99a35cbf8a6), I would suggest that this option is set somewhere around line 101 in the darwin specific build flags.

Reply:
It doesn't build on linux with alembic, since we don't have alembic in buildInputs.  IMHO we should just build with alembic on both linux and macos.

Reply:
yes, this is what I was actually checking. And as it is, alembic is disabled on linux also.

I guess it could make sense to  add support to alembic, unless there is a reason not to (I’ll compare closure sizes, but does not seem to be a problem to add alembic as a dependency).

Reply:
Without alembic support:

```
$ nix path-info -Sh /nix/store/zkg5hgzcqfwgypb60mp3wxhqiwimm29s-blender-2.82
/nix/store/zkg5hgzcqfwgypb60mp3wxhqiwimm29s-blender-2.82	477.6M
```

With alembic support:

```
$ nix path-info -Sh /nix/store/6wpcsl2annpgvpcgp5arxzsj06482pil-blender-2.82
/nix/store/6wpcsl2annpgvpcgp5arxzsj06482pil-blender-2.82	487.8M
```

Closure size is higher, but for such application I do not see that as a problem.

I would then suggest something like:

```diff
diff --git a/pkgs/applications/misc/blender/default.nix b/pkgs/applications/misc/blender/default.nix
index c14d0cda1df..18c65a912d2 100644
--- a/pkgs/applications/misc/blender/default.nix
+++ b/pkgs/applications/misc/blender/default.nix
@@ -9,6 +9,7 @@
 , colladaSupport ? true, opencollada
 , enableNumpy ? false, makeWrapper
 , pugixml, SDL, Cocoa, CoreGraphics, ForceFeedback, OpenAL, OpenGL
+, alembic
 }:
 
 with lib;
@@ -31,6 +32,7 @@ stdenv.mkDerivation rec {
     [ boost ffmpeg gettext glew ilmbase
       freetype libjpeg libpng libsamplerate libsndfile libtiff
       opencolorio openexr openimageio2 openjpeg python zlib fftw jemalloc
+      alembic
       (opensubdiv.override { inherit cudaSupport; })
       tbb
       makeWrapper
@@ -75,9 +77,7 @@ stdenv.mkDerivation rec {
     '';
 
   cmakeFlags =
-    [
-      "-DWITH_ALEMBIC=OFF"
-      "-DWITH_MOD_OCEANSIM=ON"
+    [ "-DWITH_MOD_OCEANSIM=ON"
       "-DWITH_CODEC_FFMPEG=ON"
       "-DWITH_CODEC_SNDFILE=ON"
       "-DWITH_INSTALL_PORTABLE=OFF"
```

It is also possible to make alembic support optional with an explicit option.

Reply:
```diff
index c14d0cda1df..0c7bd989d3d 100644
--- a/pkgs/applications/misc/blender/default.nix
+++ b/pkgs/applications/misc/blender/default.nix
@@ -9,6 +9,7 @@
 , colladaSupport ? true, opencollada
 , enableNumpy ? false, makeWrapper
 , pugixml, SDL, Cocoa, CoreGraphics, ForceFeedback, OpenAL, OpenGL
+, alembic
 }:
 
 with lib;
@@ -31,6 +32,7 @@ stdenv.mkDerivation rec {
     [ boost ffmpeg gettext glew ilmbase
       freetype libjpeg libpng libsamplerate libsndfile libtiff
       opencolorio openexr openimageio2 openjpeg python zlib fftw jemalloc
+      alembic
       (opensubdiv.override { inherit cudaSupport; })
       tbb
       makeWrapper
@@ -76,7 +78,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags =
     [
-      "-DWITH_ALEMBIC=OFF"
+      "-DWITH_ALEMBIC=ON"
       "-DWITH_MOD_OCEANSIM=ON"
       "-DWITH_CODEC_FFMPEG=ON"
       "-DWITH_CODEC_SNDFILE=ON"
```

to be explicit.